src/analyses/base.ml

```diff
@@ -139,6 +141,11 @@ struct
     | BNot -> ID.bitnot
     | LNot -> ID.lognot
 
+  let unop_IDB = function
```

Comment:
Push into `invariant` since it's only used there.

Reply:
Done, in b6997e81485c9e7f97903ec78eb8450e96afa6c9.